dune-project

```diff
@@ -0,0 +1,21 @@
+(lang dune 2.0)
+(generate_opam_files true)
+
+(source (github xavierleroy/cryptokit))
+(authors "Xavier Leroy")
+(maintainers "Xavier Leroy <xavier.leroy@inria.fr>")
+
+(package
+ (name cryptokit)
+ (synopsis "A library of cryptographic primitives")
+
+ (description "Cryptokit includes block ciphers (AES, DES, 3DES), stream ciphers
+(ARCfour), public-key crypto (RSA, DH), hashes (SHA-1, SHA-256,
+SHA-3), MACs, compression, random number generation -- all presented
+with a compositional, extensible interface.")
+
+ (depends
+  (ocaml (>= 4.02.0))
+  (zarith (>= 1.4))
```

Comment:
I believe you're missing dependencies on both dune and dune-configurator.

Reply:
Thanks, well spotted! Fixed.